test/dummy/app/models/validated_thing.rb

```diff
@@ -18,4 +18,26 @@ class ValidatedThing < ActiveRecord::Base
   measured_length :length_presence
   validates :length_presence, measured: true, presence: true
 
+  measured_length :length_numericality_inclusive
+  validates :length_numericality_inclusive, measured: {greater_than_or_equal_to: :low_bound, less_than_or_equal_to: :high_bound }
```

Comment:
Neat. We can probably make `Measured::Measurable.new` accept a hash as args and look for the `value: unit:`. 

The latter we intentionally avoided so we are not overloading `Fixnum` with implementation details of measurements. It's a slippery slope. Ruby Units and Unitwise do this and it's a mess. Especially the former. In fact: https://github.com/olbrich/ruby-units/issues/124